<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1341</width>
    <height>696</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RootFinder for Non-Linear Equation</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1320</width>
      <height>631</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="Bisection">
     <attribute name="title">
      <string>Bisection</string>
     </attribute>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>230</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>310</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="FalsePosition">
     <attribute name="title">
      <string>False-Position</string>
     </attribute>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>370</y>
        <width>118</width>
        <height>3</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="NewtonRaphson">
     <attribute name="title">
      <string>Newton-Raphson</string>
     </attribute>
     <widget class="QTextEdit" name="persNewton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>401</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="deriveNewton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>401</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="persamaanNewton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>151</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Persamaan</string>
      </property>
     </widget>
     <widget class="QLabel" name="turunanPersamaanNewton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>221</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Turunan Persamaan</string>
      </property>
     </widget>
     <widget class="QLabel" name="raeMaksimumNewton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>180</y>
        <width>201</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>RAE Maksimum (%)</string>
      </property>
     </widget>
     <widget class="QLabel" name="titikAwalx0Newton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>270</y>
        <width>151</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Titik Awal (x0)</string>
      </property>
     </widget>
     <widget class="QLabel" name="iterasiMaksimumNewton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>360</y>
        <width>201</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Iterasi Maksimum</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cariNewton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>510</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Cari Akar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="resetNewton">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>510</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="raeNewton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>210</y>
        <width>401</width>
        <height>41</height>
       </rect>
      </property>
      <property name="decimals">
       <number>6</number>
      </property>
      <property name="minimum">
       <double>0.000000000000000</double>
      </property>
      <property name="maximum">
       <double>100.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="x0Newton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>300</y>
        <width>401</width>
        <height>41</height>
       </rect>
      </property>
      <property name="decimals">
       <number>6</number>
      </property>
      <property name="maximum">
       <double>100.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>1.000000000000000</double>
      </property>
     </widget>
     <widget class="QSpinBox" name="iterasiNewton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>390</y>
        <width>401</width>
        <height>41</height>
       </rect>
      </property>
      <property name="maximum">
       <number>1000000</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableNewton">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>40</y>
        <width>841</width>
        <height>520</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>168</number>
      </attribute>
      <column>
       <property name="text">
        <string>x</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>x_old</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>f(x_old)</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>f'(x_old)</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>RAE (%)</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="Secant">
     <attribute name="title">
      <string>Secant</string>
     </attribute>
     <widget class="QTextEdit" name="persSecant">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>401</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="persamaanSecant">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>151</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Persamaan</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="x0Secant">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>210</y>
        <width>401</width>
        <height>41</height>
       </rect>
      </property>
      <property name="decimals">
       <number>6</number>
      </property>
      <property name="maximum">
       <double>100.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>1.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="iterasiMaksimumSecant">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>360</y>
        <width>201</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Iterasi Maksimum</string>
      </property>
     </widget>
     <widget class="QPushButton" name="resetSecant">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>510</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="raeSecant">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>401</width>
        <height>41</height>
       </rect>
      </property>
      <property name="decimals">
       <number>6</number>
      </property>
      <property name="maximum">
       <double>100.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="raeMaksimumSecant">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>201</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>RAE Maksimum (%)</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="iterasiSecant">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>390</y>
        <width>401</width>
        <height>41</height>
       </rect>
      </property>
      <property name="maximum">
       <number>1000000</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
     <widget class="QLabel" name="titikAwalx0Secant">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>180</y>
        <width>151</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Titik Awal (x0)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cariSecant">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>510</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Cari Akar</string>
      </property>
     </widget>
     <widget class="QLabel" name="titikAwalx1Secant">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>270</y>
        <width>211</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Titik Awal lain (x1)</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="x1Secant">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>300</y>
        <width>401</width>
        <height>41</height>
       </rect>
      </property>
      <property name="decimals">
       <number>6</number>
      </property>
      <property name="maximum">
       <double>100.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>1.000000000000000</double>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableSecant">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>40</y>
        <width>841</width>
        <height>520</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>140</number>
      </attribute>
      <column>
       <property name="text">
        <string>x</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>x_old_1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>x_old_2</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>f(x_old_1)</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>f(x_old_2)</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>RAE (%)</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1341</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuRootFinder_for_Non_Linear_Equation">
    <property name="title">
     <string>RootFinder for Non-Linear Equation</string>
    </property>
   </widget>
   <addaction name="menuRootFinder_for_Non_Linear_Equation"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>resetNewton</sender>
   <signal>clicked()</signal>
   <receiver>iterasiNewton</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>420</x>
     <y>642</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>519</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resetNewton</sender>
   <signal>clicked()</signal>
   <receiver>x0Newton</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>401</x>
     <y>623</y>
    </hint>
    <hint type="destinationlabel">
     <x>326</x>
     <y>427</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resetNewton</sender>
   <signal>clicked()</signal>
   <receiver>raeNewton</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>413</x>
     <y>628</y>
    </hint>
    <hint type="destinationlabel">
     <x>376</x>
     <y>350</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resetNewton</sender>
   <signal>clicked()</signal>
   <receiver>deriveNewton</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>418</x>
     <y>630</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resetNewton</sender>
   <signal>clicked()</signal>
   <receiver>persNewton</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>426</x>
     <y>629</y>
    </hint>
    <hint type="destinationlabel">
     <x>329</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resetSecant</sender>
   <signal>clicked()</signal>
   <receiver>iterasiSecant</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>397</x>
     <y>635</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>530</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resetSecant</sender>
   <signal>clicked()</signal>
   <receiver>x1Secant</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>424</x>
     <y>637</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>440</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resetSecant</sender>
   <signal>clicked()</signal>
   <receiver>x0Secant</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>411</x>
     <y>639</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>350</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resetSecant</sender>
   <signal>clicked()</signal>
   <receiver>raeSecant</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>426</x>
     <y>642</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resetSecant</sender>
   <signal>clicked()</signal>
   <receiver>persSecant</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>423</x>
     <y>623</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
